sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -24,15 +32,15 @@
 
 
 class ContainerRepositoryClient(ContainerRegistryBaseClient):
-    def __init__(self, endpoint: str, repository: str, credential: "AsyncTokenCredential", **kwargs) -> None:
+    def __init__(self, endpoint: str, repository: str, credential: "AsyncTokenCredential", **kwargs: Dict[str, Any]) -> None:
```

Comment:
I don't think `Dict[str, Any]` is wrong for **kwargs, but I think I've generally seen this documented as `**Any` in other libraries.